gammapy/data/obs_table.py

```diff
@@ -351,6 +352,41 @@ def select_observations(self, selection=None):
         else:
             raise ValueError("Invalid selection type: {}".format(selection["type"]))
 
+    def create_gti(self, obs_id):
+        """Returns a GTI table containing TSTART and TSTOP from the table.
+
+        TODO: This method can be removed once GTI tables are explicitly required in Gammapy.
+
+        Parameters
+        ----------
+        obs_id : int
+            ID of the observation for which the GTI table will be created
+
+        Returns
+        -------
+        gti : `~gammapy.data.GTI`
+            GTI table containing one row (TSTART and TSTOP of the observation with `obs_id`)
+        """
+        header = OrderedDict(
+            EXTNAME="GTI",
+            HDUCLASS="GADF",
+            HDUDOC="https://github.com/open-gamma-ray-astro/gamma-astro-data-formats",
+            HDUVERS="0.2",
+            HDUCLAS1="GTI",
+            MJDREFI=self.meta["MJDREFI"],
+            MJDREFF=self.meta["MJDREFF"],
+            TIMEUNIT=self.meta.get("TIMEUNIT", "s"),
+            TIMESYS=self.meta["TIMESYS"],
+            TIMEREF=self.meta.get("TIMEREF", "LOCAL"),
+        )
+
+        idx = self.get_obs_idx(obs_id)[0]
+        start = self["TSTART"][idx].astype("float64")
```

Comment:
I don't think an `astype("float64")` here can be helpful; the spec says it should be 64-bit, and if it isn't going 32 -> 64 at this point doesn't help, if the precision is already low, the result will be the same.

I'll do a small edit here to ensure the unit in the GTI table is "s". It should already be in the obs table, but given that you hardcode `self.meta.get("TIMEUNIT", "s")` above, I think it's a little less error-prone to also go to "s" for the value here.